src/classes/STG_InstallScript.cls

```diff
@@ -338,20 +338,21 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
     }
     
-    /**********************************************************************************************************
-    * @description Enables prevent deletion by default on Hierarchy Custom Settings for new installs and 
-    * when upgrading to 1.95
+    /**************************************************************************************************************************
+    * @description Enables prevent deletion by default on Hierarchy Custom Settings for new installs and when upgrading to 1.95
     * @return void
     */
     global static void enablePreventDeletionHierarchyCustomSettings(Version previousVersion) {
-        if ((previousVersion == NULL) || (previousVersion != null && previousVersion.compareTo(new Version(1,95)) < 0)) {
+        if ((previousVersion == NULL) || (previousVersion != NULL && previousVersion.compareTo(new Version(1,95)) < 0)) {
             UTIL_Debug.debug('***Upgrading to version 1.95 or new installs');
             Hierarchy_Settings__c orgSettings = UTIL_CustomSettingsFacade.getSettings();
-            if (orgSettings.Prevent_Plan_Requirement_Deletion__c == null || orgSettings.Prevent_Plan_Requirement_Deletion__c != true) {
-                orgSettings.Prevent_Plan_Requirement_Deletion__c = true; 
+            if (orgSettings.Prevent_Plan_Requirement_Deletion__c == NULL || orgSettings.Prevent_Plan_Requirement_Deletion__c != TRUE) {
```

Comment:
I don't think a boolean custom setting can be null thou...

Reply:
Resolving, discussed offline.